File: Examples/Node.js/SDK/manipulate/ConvertImageFormat.js
var fs = require('fs'),
    utils = require('../utils.js'),
    image = utils.imagingApi;
storage = utils.storageApi;

var inputName = "sample1.png";
var inputPath = utils.getPath(__filename, inputName);

var outputName = "nodejsSDK.nodejsSDK.output.jpg";
var outputPath = utils.getPath(__filename, outputName);

var format = 'jpeg';

image.PostImageSaveAs(format, null, inputPath, function (responseMessage) {
    var writeStream = fs.createWriteStream(outputPath);
    writeStream.write(responseMessage.body);
});

File: Examples/Python/SDK/manipulate/ConvertImageFormat.py
import common

input_file = "sample1.png"
input_path = common.get_path(__file__, input_file)

output_file = "output.jpg"
output_path = common.get_path(__file__, output_file)

format = "jpg"

# invoke Aspose.Imaging Cloud SDK API to perform format change operation without using cloud storage
response = common.imagingApi.PostImageSaveAs(format, file=input_path)

if response.Status == 'OK':
    # download image from API response
    with open(output_path, 'wb') as f:
        for chunk in response.InputStream:
            f.write(chunk)
